src/forms/StripePaymentForm/StripePaymentForm.js

```diff
@@ -1,9 +1,16 @@
+/**
+ * Note: This form is using card from Stripe Elements https://stripe.com/docs/stripe-js#elements
+ * Card is not a Final Form field so it's not available trough Final Form.
+ * It's also handled separately in handleSubmit function.
+ */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { Form as FinalForm } from 'react-final-form';
 import classNames from 'classnames';
-import { Form, PrimaryButton, ExpandingTextarea } from '../../components';
+import { Form, PrimaryButton, FieldTextInput } from '../../components';
```

Comment:
Reorder these so that components are after config and util files:
// external libs
// config & utils
// shared components

// css & local subcomponents from this directory